router-map/static/js/connection-fail.js

```diff
@@ -0,0 +1,39 @@
+import toastr from 'toastr';
+import 'toastr/build/toastr.css';
+
+const connection_timeout = 1000
```

Comment:
```suggestion
const CONNECTION_TIMEOUT = 1000
```
It's good practice to write constants variable names in capital letters